Content.Shared/_RMC14/Xenonids/Construction/XenoConstructionComponent.cs

```diff
@@ -42,4 +42,7 @@ public sealed partial class XenoConstructionComponent : Component
     {
         Params = AudioParams.Default.WithVolume(-10f)
     };
+
+    [DataField, AutoNetworkedField]
+    public bool IsSlowAnimation = false;
```

Comment:
this is like the worst implematation ever, i was gonna make construction use prorotypes and have an animation field

hardcoding slow animations to 4 seconds is bad

Reply:
I did feel liek there was a better way... I'll look at what smug is talking about